lib/Sema/LifetimeDependence.cpp

```diff
@@ -391,12 +385,11 @@ std::optional<LifetimeDependenceInfo> LifetimeDependenceInfo::fromTypeRepr(
           : nullptr,
       scopeLifetimeParamIndices.any()
           ? IndexSubset::get(ctx, scopeLifetimeParamIndices)
-          : nullptr,
-      /*isExplicit*/ true);
+          : nullptr);
 }
 
 std::optional<LifetimeDependenceInfo>
-LifetimeDependenceInfo::infer(AbstractFunctionDecl *afd, Type resultType) {
+LifetimeDependenceInfo::infer(AbstractFunctionDecl *afd) {
```

Comment:
In a subsequent PR, please refactor this logic so that LifetimeDependence methods are implemented in AST. Perhaps these functions can just be static, if they're only used in the implementation of the request evaluate() method?

Reply:
Will do. Can change them to static